src/test/java/CartTests.java

```diff
@@ -0,0 +1,67 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Nested
```

Comment:
what's the reason to use "@Nested" annotation? try to run code without it

Reply:
It's just for task "Tests should be grouped in different categories."

Reply:
I got it, I used @Tag annotation.